frank/utilities.py

```diff
@@ -218,6 +222,10 @@ def cut_data_by_baseline(u, v, vis, weights, cut_range):
         Weights in the chosen baseline range
     """
 
+    logging.info('  Cutting data outside of the minmum and maximum baselines'
+                 ' of {} and {}'
+                 ' klambda'.format(cut_range[0] / 1e3, cut_range[1] / 1e3))
+
```

Comment:
I genuinely think this is unnecessary. If you *really* want to log this function, do it at the debug level, not info

Reply:
I feel the same about this one as load_uvtable

Reply:
Moved to fit.py